scripts/script.js

```diff
@@ -108,23 +106,25 @@ cocktailApp.displayDrinks = function (data) {
 
     // Add an event listener and append to the page
     listElement.addEventListener('click', cocktailApp.handleCardClick);
-    cocktailApp.drinkList.append(listElement);
+    cocktailApp.drinksList.append(listElement);
   });
 
   // carousel function
   // giving the first child element (li) a class of "drink__pic--current"
-  cocktailApp.drinkList.children[0].classList.add('drink__pic--current');
+  const firstDrink = cocktailApp.drinksList.children[0]
+  firstDrink.classList.add('drink__pic--current');
+  cocktailApp.changeDrinkSectionHeight(firstDrink);
 
-  buttons.forEach(button => {
+  cocktailApp.carouselButtons.forEach(button => {
```

Comment:
I was actually going to ask you about this!
Thanks for putting it into so much better namespacing!